### ISSUE TITLE
Everyone's book

### ISSUE THREAD

**Issue by doubleshow** [bigdatahci2015/forum#13]
We've created a page that lists everyone's learning book.

http://bigdatahci2015.github.io/course/learning/books/index.html

Please take a moment to ensure that your book is published properly, because the grader and myself will start looking at them soon. Click on your _week1_ and _week2_ links and make sure you do not get a _404 Page not found error_.

**Comment by pillig** [comment]
Just discovered my week 1 svg template didn't get built into the book despite having the .md~ file. When are you going to be looking at the books exactly so I can fix it before then?

**Comment by KevinKGifford** [comment]
I am unable to publish my Week 2 submission (or Week 3) due to git push conflicts that I've been unable to resolve.  Is there some type of assistance I can receive to resolve these issues so can submit exercises prior to due date (offices hours or e-mail exchange)?

**Comment by doubleshow** [comment]
@KevinKGifford a solution that might work is to go to your book's branches page on Github
https://github.com/KevinKGifford/book/branches
Click on the the delete button associated with the _gh-pages_ branch. This may remove the cause of the conflict. Then, on your local machine, try to push subtree again and see if the conflict goes away.

Please confirm whether this works. If not, let's schedule a time on Monday before the class to resolve this issue together.

**Comment by doubleshow** [comment]
@pail4944 please fix them before our Monday meeting. Thanks!

**Comment by KevinKGifford** [comment]
Yes, this worked.  My issue made me review git in detail which is a good
thing.  Thank you for the tip/assistance, very much appreciated.

Kevin
